src/v4l2_camera.cpp

```diff
@@ -48,6 +48,15 @@ V4L2Camera::V4L2Camera(rclcpp::NodeOptions const & options)
   // This should happen before registering on_set_parameters_callback,
   // else transport plugins will fail to declare their parameters
   bool use_sensor_data_qos = declare_parameter("use_sensor_data_qos", false);
+  publish_rate_ = declare_parameter("publish_rate", 10.0);
```

Comment:
```suggestion
  publish_rate_ = declare_parameter("publish_rate", -1.0);
```
Since I guess controlling publish rate is an optional function, how about turning off the function by default?